src/plots/gl2d/scene2d.js

```diff
@@ -318,6 +318,14 @@ proto.destroy = function() {
 
     this.glplot = null;
     this.stopped = true;
+
+    var traces = this.traces;
+    if(traces) {
```

Comment:
Nice find. 

I think the `gl-plot3d` cleans its trace objects internally, but it would worth double checking :+1: